valohai_cli/commands/execution/ssh.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import click
+
+from valohai_cli.exceptions import CLIException
+from valohai_cli.utils.cli_utils import counter_argument
+from valohai_cli.utils.ssh import (
+    get_ssh_details_with_retry,
+    make_ssh_connection_to_container,
+    select_private_key_from_possible_directories,
+)
+
+
+@click.command()
+@counter_argument
+@click.option("--private-key-file", default=None, help="Private SSH key to use for the connection.")
```

Comment:
```suggestion
@click.option("--private-key-file", default=None, type=click.Path(file_okay=True, exists=True), help="Private SSH key to use for the connection.")
```
and you don't need to validate the existence of the file by hand.